src/bbtautau/postprocessing/SensitivityStudy.py

```diff
@@ -527,7 +619,20 @@ def sig_bg(bbcut, ttcut):
                 mtt2=mtt2,
```

Comment:
It looks like you're applying mass cuts on the ττ jet for the BDT optimization too? You can remove that for the BDT, since it takes the ττ mass as input.